eth/block-validation/api_test.go

```diff
@@ -845,6 +845,99 @@ func executableDataToBlockValidationRequest(execData *engine.ExecutableData, pro
 	return blockRequest, nil
 }
 
+func TestValidateBuilderSubmissionV2_CoinbasePaymentUnderflow(t *testing.T) {
```

Comment:
should use the v3 builder submission which is currently being used, though ideally the unit test should just be calling `ValidatePayload`